scripts/pypi_release.sh

```
@@ -0,0 +1,16 @@
+set -e
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)